sandbox/sandbox_json.cpp

```diff
@@ -320,13 +320,13 @@ enum Bla
 template <class Archive>
 void save( Archive & ar, Bla const & b )
 {
-  ar( (const int &)b );
+  ar( static_cast<const int &>(b) );
```

Comment:
I don't totally see what is happening here, but is the cast totally necessary?  Should this cast be to std::underlying_type?